azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -928,7 +928,7 @@ public void cancelExecutingJobs(ExecutableFlow exFlow, String userId,
     }
   }
 
-  private void applyDisabledJobs(List<Object> disabledJobs,
+  public static void applyDisabledJobs(List<Object> disabledJobs,
```

Comment:
If that's the case, then they should be moved to a util class. `Utils` in general is a tricky thing in terms of coding practices. http://stackoverflow.com/questions/3340032/utility-classes-are-evil